src/shouldFail.js

```diff
@@ -1,4 +1,5 @@
 const { should } = require('./setup');
+const c = require('ansi-colors');
```

Comment:
I know this is what they suggest, but having `c` be a variable containing a module isn't the best idea IMO. Could we rename this to something like `colors`?